readme.txt

```diff
@@ -152,22 +152,15 @@ See [labeled GitHub issues](https://github.com/xwp/pwa-wp/issues?q=label%3Aservi
 = Integrations =
 The plugin bundles several experimental integrations that are kept separate from the service worker core code. These integrations act as examples and proof-of-concept to achieve certain goals. While all of them are generally applicable and recommended to truly benefit from service workers, they are not crucial for the core API.
 
-All these integrations are hidden behind a feature flag. To enable them, you can either set a constant `WP_SERVICE_WORKER_INTEGRATIONS_ENABLED` in your `wp-config.php` and set it to true, or add a one-liner must-use plugin:
+All these integrations are hidden behind a feature flag. To enable them, you can add `service_worker` theme support:
 
 <pre lang="php">
 <?php
-/*
-Plugin Name: Enable Service Worker Integrations
-Plugin URI:  https://github.com/xwp/pwa-wp
-Description: Enables the experimental service worker integrations in the PWA plugin.
-Version:     1.0.0
-Author:      XWP, Google, and contributors
-Author URI:  https://github.com/xwp/pwa-wp
-*/
-
-add_filter( 'wp_service_worker_integrations_enabled', '__return_true' );
+add_theme_support( 'service_worker', true );
```

Comment:
@felixarntz should this be updated to include the array syntax?

Reply:
Added an extra example to the readme.